src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver() {
```

Comment:
I think we should define toString(). method here (to display info about the driver in exception messages)